src/analysis/convex/cone.lean

```diff
@@ -189,44 +189,20 @@ ext' $ preimage_comp.symm
 @[simp] lemma mem_comap {f : E →ₗ[ℝ] F} {S : convex_cone F} {x : E} :
   x ∈ S.comap f ↔ f x ∈ S := iff.rfl
 
+-- We use `{ to_semimodule := ‹_›, .. }` to avoid making this definition noncomputable
```

Comment:
How can I improve it? If I write `:= ordered_semimodule.mk' _`, then Lean thinks that the definition is noncomputable. We can avoid this by taking only proof fields from `ordered_semimodule.mk' _`. I'm not sure that it's worth it because there are no computable (in the sense "useful for computation", not in the sense `computable`) modules over `real` anyway.